src/app/panels/text-panel/text-panel.component.ts

```diff
@@ -50,13 +51,31 @@ export class TextPanelComponent implements OnDestroy {
   private showSecondaryContent = false;
 
   public selectedPage;
+
+  public textFlow: TextFlow = AppConfig.evtSettings.edition.defaultTextFlow || 'prose';
+  public enableProseVersesToggler = AppConfig.evtSettings.edition.proseVersesToggler;
+  public get proseVersesTogglerIcon(): EvtIconInfo {
+    return this.textFlow === 'prose' ? { icon: 'align-left', iconSet: 'fas' } : { icon: 'align-justify', iconSet: 'fas' };
```

Comment:
what about `return { icon: this.textFlow === 'prose' ? 'align-left' : `align-justify', iconSet: 'fas' }`?